app/src/main/java/com/pvp/app/ui/screen/profile/ProfileScreen.kt

```diff
@@ -266,124 +434,389 @@ private fun UserInfoItem(
             ) {
                 IconButtonWithDialog(
                     icon = Icons.Default.Edit,
```

Comment:
Use `Outlined` icons everywhere please. To stay consistent. `Icons.Outlined.Edit`. I may have missed more spots like this one, make sure it is followed everywhere throughout this file and Filter file.